app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,6 @@
   <head>
     <title>Quibble</title>
     <%= stylesheet_link_tag    "application", media: "all" %>
-    <%= stylesheet_link_tag    "materialize", media: "screen, projection" %>
```

Comment:
Yes, I believe that the application.css takes care of including this.  require tree....i believe